pkg/exec/exec_test.go

```diff
@@ -8,8 +8,8 @@ import (
 	"testing"
```

Comment:
the breaking change only appeared in the main business logic, not in the `pkg` folder. So can we revert the `pkg/` folder changes?

Reply:
its not able to understand the import path without the new changes as essentially the module name is changed